blog/2023-08-10-evaluating-open-source-llms/index.md

```diff
@@ -0,0 +1,136 @@
+---
+slug: evaluating-open-source-llms
+title: Evaluating Open-Source Large Language Models
+authors: [het]
+tags: [llm, prem, performance, dataset]
+description: "Understand how the performance of large language models is evaluated"
+image: "./banner.jpeg"
+---
+<!--truncate-->
+
+![Prem Banner](./banner.jpeg)
+
+<head>
+  <meta name="twitter:image" content="./banner.png"/>
+</head>
+
+## Introduction
+
+By now, it seems like every month a new open-source Large Language Model(LLM) comes along and breaks all of the records held by previous models. 
+
+The pace at which generative AI is progressing is so quick that people are spending most of their time catching up rather than building useful tools. There is a lot of confusion in the open-source community as to which LLM is the best. Businesses want to use the best open-source LLM for their use case. But how do you know which one is the best?
+
+```txt
+Person 1: You know, I've been thinking, we should definitely consider using one of these large language models for our project.
+Person 2: True, but which one should we go for?
+Person 3: Maybe we should try MPT! It's known for its amazing fluency and coherence in generated text.
+Person 1: Yeah, but wait, Falcon looks cool too. It claims to handle a wide range of language tasks effortlessly.
+Person 2: And LLama is available for commercial use, so that's something to consider.
+```
+
+## How LLMs Are Benchmarked
+
+Most people that play around with LLMs can tell how well a model performs just by the output they're getting. But how can you numerically measure an LLM's performance?
+
+Currently, the way LLMs are benchmarked is by testing them on a variety of datasets. The [hugging face leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard) has a nice visual representation of how well open-source LLMs perform on 4 standard datasets and where they rank in the leaderboard. 
```

Comment:
link looks borked?

![image](https://github.com/premAI-io/dev-portal/assets/10780059/810d04dd-5070-40e4-b110-30a2af5b4001)

Reply:
I think the link should work now. Maybe the hugging face leaderboard was down when you tried it?

Reply:
yes working now. Heisenbug :)